index.html

```diff
@@ -574,6 +574,39 @@ <h3>Cheat Sheet:</h3>
           >
         </p>
       </div>
+      <div class="emoji">
+        <div class="miltonRutus" style="margin-top: 1px">
+          <div class="beer30">
+            <div class="bubles">
+                <div class="b1"></div>
+                <div class="b2"></div>
+                <div class="b3"></div>
+                <div class="b4"></div>
+                <div class="b5"></div>
+                <div class="b6"></div>
+                <div class="b7"></div>
+                <div class="b8"></div>
+                <div class="b9"></div>
+             </div>
+          <div class="beer3"></div>
+          <div class="beer">
+            <div class="ojos"><div class="iris"></div></div>
+            <div class="ojos3"><div class="iris"></div></div>
+            <div class="boca"></div>
+            <div class="line"></div>
+            <div class="rastro"></div>
```

Comment:
Hello, great work! Your html should only have one div though. The `div` with class `emoji` should only have one child, your id :)